piet-wgsl/src/render.rs

```diff
@@ -52,7 +53,14 @@ fn size_to_words(byte_size: usize) -> u32 {
     (byte_size / std::mem::size_of::<u32>()) as u32
 }
 
-pub fn render(scene: &Scene, shaders: &Shaders) -> (Recording, BufProxy) {
+pub const fn next_multiple_of(val: u32, rhs: u32) -> u32 {
+    match val % rhs {
```

Comment:
One other way to write this is `val + rhs - 1 - (val + rhs - 1) % rhs`

Reply:
I copied this from the unstable `std` implementation, and this is not hot

I'd be inclined to leave it as it.